submissions/annStoyano/html-css-popup/scss/style.scss

```diff
@@ -0,0 +1,263 @@
+    height: 30px;
+}
+
+.popup_btn-icon {
+    position: absolute;
+    background: url('../popup-icons/popup-button.png') center no-repeat;
+    width: 30px;
+    height: 30px;
+
+    &:hover {
+        cursor: pointer;
+    }
+}
+
+.popup_btn {
+    position: absolute;
+    opacity: 0;
+}
+
+.popup_btn-icon:hover,
+.popup_btn:focus+.popup_btn-icon {
+    background-color: rgba(218, 218, 218, 0.3);
+    border-radius: 50%;
+}
+
+.popup_btn:checked~.popup {
+    display: block;
+}
+
+.popup {
+    display: none;
+    border: 1px solid black;
+    width: 310px;
+    height: 500px;
+    position: absolute;
+    top: 160%;
+    left: -200px;
+
+    /*Triangle*/
+    &::before {
+        content: '';
+        position: absolute;
+        width: 0;
+        height: 0;
+        border-left: 16px solid transparent;
+        border-right: 16px solid transparent;
+        border-bottom: 16px solid black;
+        z-index: -1;
+        top: -16px;
+        left: 198px;
+    }
+
+    /*Inner Triangle (Make tringle border for popup box)*/
+    &::after {
+        content: '';
+        position: absolute;
+        width: 0;
+        height: 0;
+        border-left: 15px solid transparent;
+        border-right: 15px solid transparent;
+        border-bottom: 15px solid white;
+        top: -15px;
+        left: 199px;
+        z-index: 5;
+    }
+
+}
+
+.popup-content{
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+}
+
+.main-menu{
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    padding: 40px 20px 5px 20px;
+    width: 100%;
+    height: 90%;
+}
+
+.popup-item:hover, .popup-item:focus{
+    outline-style: none;
+    border: 1px solid gray;
+}
+
+.popup-item {
+    width: 80px;
+    height: 80px;
+    padding: 3px;
+    display: flex;
+    justify-content: center;
+    text-align: center;
+
+    &-icon {
+        width: 100%;
+        height: 100%;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+
+        img {
+            margin: 0 auto;
+            max-width: 55px;
+            max-height: 45px;
+            object-fit: cover;
+        }
+    }
+}
+
+.show-more_btn{
+    opacity: 0;
+    position: absolute;
+}
+
+.show-more:hover,.show-more_btn:focus + .show-more{
+    color: white;
+    background-color:  rgba(56, 56, 56, 0.733);
+    cursor: pointer;
+}
+
+.show-more{
+    transition: all 0.2s ease-in;
+    padding: 7% 44.4%;
+    position: absolute;
+    bottom:0%;
+    background-color:  rgba(218, 218, 218, 0.3);
+}
+
+.show-more_btn:checked ~ .sub-menu{
+    display: flex;
+    border-top: 1px solid  rgb(218, 218, 218);;
+}
+
+.show-more_btn:checked + .show-more{
+    display: none;
+}
+
+.sub-menu{
+    display: none;
+    justify-content: space-between;
+    align-content: space-between;
+    flex-wrap: wrap;
+    padding: 40px 20px 10px 20px;
+    width: 100%;
+    height: 55%;
+}
+
+.kottans{
+    margin: 10px auto 0px auto;
+}
```

Comment:
Don't forget about a new line.

You can also set your text editor for adding a new empty line